### PR TITLE
WT-9975 Log stats for c and format tests

### PR DESCRIPTION
**Pull request by quchenhao**

### CODE REVIEW

File: bench/wt2853_perf/main.c [hunk 1]
```diff
@@ -107,8 +107,8 @@ main(int argc, char *argv[])
     testutil_make_work_dir(opts->home);
     testutil_progress(opts, "start");
 
-    testutil_check(
-      wiredtiger_open(opts->home, NULL, "create,cache_size=1G,statistics=(all)", &opts->conn));
+    testutil_check(wiredtiger_open(opts->home, NULL,
+      "create,cache_size=1G,statistics=(all),statistics_log=(json,on_close,wait=5)", &opts->conn));
```
Comment (etienneptl) [inline bench/wt2853_perf/main.c:111]: Is `5` on purpose? The other changes use `1`.

Comment (quchenhao) [reply bench/wt2853_perf/main.c:111]: perf test should have a longer interval I think.

Comment (quchenhao) [reply bench/wt2853_perf/main.c:111]: test format is also set as 5

**Review by etienneptl — APPROVED** [review]
LGTM, thanks!

**Review by korteland — APPROVED** [review]
LGTM - sorry for taking so long to get around to the review.

**Comment by quchenhao** [comment]
evergreen merge

**Pull request merged**